swan.sh

```diff
@@ -1,44 +0,0 @@
-package: SWAN
```

Comment:
The reason why I had a separate package for the swan modules is that doing it as part of the Python-modules will force a rebuild of a bunch of generators for no particular good reason. Is this actually something we want? If yes, we should be extremely careful when merging in production and announce it properly to the various generators users.